Update error-response-codes.md (#2510)
TykTechnologies/tyk-docs: tyk-docs/content/error-response-codes.md

- 5xx (Server Error): The server failed to fulfill an apparently valid request

## Tyk Gateway error status code

Below is a list of error status codes (4xx and 5xx) along with their corresponding messages that may be returned by the Tyk gateway. 

> If you require customised message for certain HTTP error codes, Tyk also supports [error templating]({{< ref "advanced-configuration/error-templates" >}}).

Reviewer: ```suggestion
Tyk supports [error templating]({{< ref "advanced-configuration/error-templates" >}}), allowing you to configure the Gateway to return customised messages for certain HTTP error codes.

We also support limited customisation of the error codes and messages returned by custom authentication middleware through the use of [override messages]({{< ref "tyk-oss-gateway/configuration#override_messages" >}}.
```

> | Code | Text                                                                                    | Description                                                                                                                                                             |

Reviewer: ```suggestion
| Code | Text                                                                                    | Recommended action                                                                                                                                                             |
```

Author: good one!!! wanted to change this as well

- 5xx (Server Error): The server failed to fulfill an apparently valid request

## Tyk Gateway error status code

Here we provide a list of all the error status codes (4xx and 5xx) that may be returned by the Tyk Gateway along with their corresponding messages and some guidance on the likely cause of the error. 
Tyk supports [error templating]({{< ref "advanced-configuration/error-templates" >}}), allowing you to configure the Gateway to return customised messages for certain HTTP error codes.

> We also support limited customisation of the error codes and messages returned by custom authentication middleware through the use of [override messages]({{< ref "tyk-oss-gateway/configuration#override_messages" >}}.

Reviewer: ```suggestion
We also support limited customisation of the error codes and messages returned by custom authentication middleware through the use of [override messages]({{< ref "tyk-oss-gateway/configuration#override_messages" >}}).
```
